### DISCUSSION TITLE
Ways to check for updates to alias dependencies

### DISCUSSION THREAD

**Discussion by robin-a-meade** [jbangdev/jbang#1751] [Ideas] [answered]
With maven projects, I like to use the `versions:display-dependency-updates` goal to see what updates are available. I also like how github's dependabot can send notifications about any out-dated dependencies it finds in a `pom.xml` file.

So, I thought maybe I could write a script that reads the alias definitions in the catalog and generates a`pom.xml` file for each. 

I'd then be able to do:
```
mvn -f <location of generated pom file> versions:display-dependency-updates
```

And if I commit the generated `pom.xml` files to the catalog repo, and configure `dependabot.yml` appropriately, I'd get notifications from dependabot.

I created a catalog repo to play around with this idea:

https://github.com/robin-a-meade/jbang-catalog

Another idea would be to teach dependabot about the jbang "package ecosystem" so that dependabot would read `jbang-catalog.json` files and understand the dependencies.

**Comment by quintesse** [comment]
That's really interesting! I think @maxandersen already has something working more or less for dependabot (or was it renovate?) that can check catalog files, but right now I can't find it.

**Comment by maxandersen** [comment, marked as answer]
Dependabot unfortunately seem to have shut down for extending their support for other approaches. 

Renovate on the other hand can be "educated". 

Try run: 'jbang init -t jbangcatalog@jbanghub mycatalog' and you'll get a project with a renovate.json that will find updates in //Deps and catalog script refs.

**Reply by robin-a-meade** [reply]
Thanks. It works!

I did:
```
jbang init -t jbanghub@jbanghub mycatalog
```
and then copied its `.github/renovate.json` file into my `jbang-catalog` repo and installed the Renovate app.

Works great!
